wsmaster/che-core-api-factory-github/src/test/java/org/eclipse/che/api/factory/server/github/GithubFileContentProviderTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
```

Comment:
new file, with old copyright. Seems wrong to me. :) but will the license checker plugin allow a file with 2021 copyright date?